cloud/amazon/ec2_elb_lb.py

```diff
@@ -360,6 +360,29 @@
 except ImportError:
     HAS_BOTO = False
 
+import time
+import random
+
+def _throttleable_operation(max_retries):
+    def _operation_wrapper(op):
+        def _do_op(*args, **kwargs):
+            retry = 0
+            while True:
+                try:
+                    return op(*args, **kwargs)
+                except boto.exception.BotoServerError, e:
```

Comment:
Yep. [You're right](http://docs.ansible.com/ansible/intro_installation.html#managed-node-requirements). Sorry!